### PR TITLE
Reskin/mnemonic phrase css

### PR DESCRIPTION
**Pull request by piyalbasu**

### CODE REVIEW

File: extension/src/popup/components/Onboarding/styles.scss [hunk 1]
```diff
@@ -2,3 +2,25 @@
...
+  max-width: 57rem;
+  width: 100%;
+  margin: auto;
+}
+
+.OnboardingHalfScreen {
+  display: flex;
+  padding: 2rem 0 2rem 1.55rem;
+  width: 27rem;
+}
```
Comment (piyalbasu) [inline extension/src/popup/components/Onboarding/styles.scss:26]: Adding these allows us to omit some duplicative styles and makes sure all the Onboarding screen stay consistent

File: extension/src/popup/components/Onboarding/index.tsx [hunk 2]
```diff
@@ -7,18 +7,67 @@ import "./styles.scss";
 
 export const Onboarding = ({
   goBack,
+  hasGoBackBtn,
   children,
 }: {
   goBack?: () => void;
```
Comment (acharb) [inline extension/src/popup/components/Onboarding/index.tsx:13]: 💭  could we add a comment here saying something like 
`// if null defaults to the BackButton goback`

took me a minute to figure out that was happening (and why both a hasGoBackBtn and a goBack var are needed)

**Review by acharb — APPROVED** [review]
lgtm! had 1 thought

File: extension/src/popup/components/Onboarding/index.tsx (hunk 2 above)
Comment (piyalbasu) [reply extension/src/popup/components/Onboarding/index.tsx:13]: Good callout. This is not clear. I'll fix this

**Review by acharb — APPROVED** [review]
lgtm!

**Pull request merged**